plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -780,6 +780,24 @@ public void testRowLevelDeletesWithTableChangesFunction()
         }
     }
 
+    @Test
+    public void testCreateOrReplaceWithTableChangesFunction()
+    {
+        try (TestTable table = new TestTable(
+                getQueryRunner()::execute,
+                "test_table_changes_function_",
+                "AS SELECT nationkey, name FROM tpch.tiny.nation WITH NO DATA")) {
+            long initialSnapshot = getMostRecentSnapshotId(table.getName());
+            assertUpdate("CREATE OR REPLACE TABLE " + table.getName() + " AS SELECT nationkey, name FROM nation", 25);
```

Comment:
How about adding additional operations before & after `CREATE OR REPLACE` statement? e.g. 
1. CREATE TABLE AS SELECT with TestTable
2. INSERT
3. CREATE OR REPLACE TABLE
4. INSERT

For verifying the query success of `table_changes` table function with 1→2 & 3→4.